### PR TITLE
BIP85: update status to Final

### PR DESCRIPTION
**Pull request by scgbckbone**
Moving this BIP to final stage as this is widely deployed

### CODE REVIEW

File: bip-0085.mediawiki [hunk 1]
```diff
@@ -5,7 +5,7 @@
   Author: Ethan Kosakovsky <ethankosakovsky@protonmail.com>
   Comments-Summary: No comments yet.
   Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0085
-  Status: Draft
+  Status: Final
```
Comment (jonatack) [inline bip-0085.mediawiki:8]: The CI is red because the `README.md` file needs to be updated as well.

**Comment by jonatack** [comment]
For context, this change was recommended yesterday in https://github.com/bitcoin/bips/pull/1600#issuecomment-2394691135.

BIP author @ethankosakovsky has been unresponsive for a long time but would be good to see community feedback here.

**Comment by jonatack** [comment]
ACK b0125501f824804dc5f39f143c3c674f3eac6c7f

For context, this change was recommended yesterday in https://github.com/bitcoin/bips/pull/1600#issuecomment-2394691135.

BIP author @ethankosakovsky has been unresponsive for a long time but would be good to see community feedback here.

**Comment by nvk** [comment]
ACK

**Comment by Rob1Ham** [comment]
ACK

**Comment by RandyMcMillan** [comment]
ACK b0125501f824804dc5f39f143c3c674f3eac6c7f

**Comment by luisschwab** [comment]
ACK b0125501f824804dc5f39f143c3c674f3eac6c7f

File: README.mediawiki [hunk 2]
```diff
@@ -448,13 +448,13 @@ Those proposing changes should consider that ultimately consent may rest with th
...
 | Final
-|-
+|- style="background-color: #cfffcf"
 | [[bip-0085.mediawiki|85]]
 | Applications
 | Deterministic Entropy From BIP32 Keychains
 | Ethan Kosakovsky
 | Informational
-| Draft
+| Final
```
Comment (akarve) [inline README.mediawiki:457]: Why Final and not Proposed per [this comment](https://github.com/bitcoin/bips/pull/1600#issuecomment-2394691135)?

Comment (scgbckbone) [reply README.mediawiki:457]: comment states "propsed or final", this BIP should be final as it is widely deployed and used by many projects.

**Comment by jonatack** [comment]
Waiting for #1679 outcome before merge here.

**Review by Princeprince559 — APPROVED** [review]

**Review by Aldocapurro — APPROVED** [review]

**Pull request merged**